### ISSUE TITLE
Eror caused by sink("NUL")

### ISSUE THREAD

**Issue by oscardelama** [AnotherSamWilson/ParBayesianOptimization#3]
Hi Sam.

I was trying to learn the basics of your package by implementing the example code in your package Readme file. The BayesianOptimization starts but few seconds later, a seemingly silly error, aborts it. The culprit is sink("NUL") (line 241):

```
Error in file(file, if (append) "a" else "w") : cannot open the connection
3. file(file, if (append) "a" else "w")
2. sink("NUL")
1. BayesianOptimization(FUN = xgb_optim, bounds = bounds, initPoints = 9, bulkNew = 1, nIters = 200, kern = kern, acq = acq, kappa = 2.576, verbose = 1, parallel = FALSE)
```

Reading [this thread](http://r.789695.n4.nabble.com/Suppressing-output-e-g-from-cat-td859876.html) (between Hadley Wickham & Brian Ripley nothing less!) I found that you can use:
`  sink("/dev/null") `
but it doesn't work in  Windows. In Windows you can use:
`  sink("nul:") `
but that works only in Windows. A cross-OS solutions might be:
`  sink(tempfile())`

By the way I am using "Linux Mint 19.1 Tessa Xfce" and a fresh installation of R:
```
R version 3.5.3 (2019-03-11) -- "Great Truth"
Copyright (C) 2019 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)
```

I am eager to try your package for job related tasks!

Cheers.

**Comment by AnotherSamWilson** [comment]
Interesting, that function does not cause a failure on the other Linux operating systems:
https://cran.r-project.org/web/checks/check_results_ParBayesianOptimization.html

I'll look into sink(tempfile()) and let you know if it works!

**Comment by AnotherSamWilson** [comment]
Fixed in commit ce6ab38445f121898f5791d87508161826d3c67c. Messages from scoring function now sink into an anonymous file.
